ui/lib/apps/Statement/utils/tableColumns.tsx

```diff
@@ -10,31 +10,81 @@ import { orange, red } from '@ant-design/colors'
 import { StatementModel } from '@lib/client'
 import { Bar, Pre } from '@lib/components'
 import {
-  TableColumnFactory,
   formatVal,
+  genDerivedBarSources,
   IColumnWithSourceFields,
+  TableColumnFactory,
 } from '@lib/utils/tableColumnFactory'
 
 ///////////////////////////////////////
 // statements order list in local by fieldName of IColumn
 // slow query order list in backend by key of IColumn
 const TRANS_KEY_PREFIX = 'statement.fields'
 
+export const derivedFields = {
+  avg_latency: genDerivedBarSources(
+    'avg_latency',
+    'max_latency',
+    'min_latency'
+  ),
+  parse_latency: genDerivedBarSources('avg_parse_latency', 'max_parse_latency'),
+  compile_latency: genDerivedBarSources(
+    'avg_compile_latency',
+    'max_compile_latency'
+  ),
+  process_time: genDerivedBarSources(
+    'avg_cop_process_time',
+    'max_cop_process_time'
+  ),
+  wait_time: genDerivedBarSources('avg_cop_wait_time', 'max_cop_wait_time'),
+  total_process_time: genDerivedBarSources(
+    'avg_process_time',
+    'max_process_time'
+  ),
+  total_wait_time: genDerivedBarSources('avg_wait_time', 'max_wait_time'),
+  backoff_time: genDerivedBarSources('avg_backoff_time', 'max_backoff_time'),
+  avg_write_keys: genDerivedBarSources('avg_write_keys', 'max_write_keys'),
+  avg_processed_keys: genDerivedBarSources(
+    'avg_processed_keys',
+    'max_processed_keys'
+  ),
+  avg_total_keys: genDerivedBarSources('avg_total_keys', 'max_total_keys'),
+  prewrite_time: genDerivedBarSources('avg_prewrite_time', 'max_prewrite_time'),
+  commit_time: genDerivedBarSources('avg_commit_time', 'max_commit_time'),
+  get_commit_ts_time: genDerivedBarSources(
+    'avg_get_commit_ts_time',
+    'max_get_commit_ts_time'
+  ),
+  commit_backoff_time: genDerivedBarSources(
+    'avg_commit_backoff_time',
+    'max_commit_backoff_time'
+  ),
+  resolve_lock_time: genDerivedBarSources(
+    'avg_resolve_lock_time',
+    'max_resolve_lock_time'
+  ),
+  local_latch_wait_time: genDerivedBarSources(
+    'avg_local_latch_wait_time',
+    'max_local_latch_wait_time'
+  ),
+  avg_write_size: genDerivedBarSources('avg_write_size', 'max_write_size'),
+  avg_prewrite_regions: genDerivedBarSources(
+    'avg_prewrite_regions',
+    'max_prewrite_regions'
+  ),
+  avg_txn_retry: genDerivedBarSources('avg_txn_retry', 'max_txn_retry'),
+  avg_mem: genDerivedBarSources('avg_mem', 'max_mem'),
+  sum_errors: ['sum_errors', 'sum_warnings'],
+  related_schemas: ['table_names'],
+}
```

Comment:
In fact, this solution is a bit similar to the old version that was replaced by bringing `sourceFields` and `IColumnWithSourceFields`, aka put the ref columns in an individual object, but this is better than before by reusing it with `statementColumns()` or `slowQueryColumns()`. So I think it's fine.

@breeswish how do you think about it?